### DISCUSSION TITLE
Nav pane width

### DISCUSSION THREAD

**Discussion by masaccio** [danirus/sphinx-nefertiti#18] [Q&A] [answered]
Is there a way to change the proportions of the nav pane? In my [project](https://masaccio.github.io/numbers-parser/) the pane is too narrow and results in a lot of line wrapping.

**Comment by danirus** [comment, marked as answer]
Yes, you could add a `custom.css` file to your project with the CSS that would change the proportions of the `grid-template-columns` used here. By default their value are: `4fr 1fr`. The width of the central area is `4fr` and the right side panel width is `1fr`, but you could do something like:

    .nftt-layout .nftt-main {
        grid-template-columns: 4fr 1.5fr;
    }

If you put that little CSS code in a `_static/custom.css`, and modify your `conf.py` to include it, like this:

    html_style = ["custom.css",]

Then it should work. Let me know :-)

**Comment by masaccio** [comment]
Perfect, thanks.

**Comment by masaccio** [comment]
Related second question - can the table of contents sidebar be changed/hidden?

**Reply by danirus** [reply]
Yes.
In the document where you want it to be hidden, put `:hidetoc: 1` at the top of the file. 
See it in the line 6 of the [index.rst](https://github.com/danirus/sphinx-nefertiti/blob/main/docs/source/index.rst?plain=1#L6), of Nefertiti for Sphinx documentation.
It works in both `.rst` and `.md` (with MyST-Parser).

**Comment by masaccio** [comment]
I have my terminology wrong then - I mean the left-hand pane with the filter box in it.

**Reply by danirus** [reply]
No, that one can't be removed. 
It is automatically hidden when using a smaller screen.

**Reply by danirus** [reply]
Hi @masaccio, in the last version, 0.3.2, I implemented what you mentioned here. The removal of the left sidebar. In the same way that we can use the `:hidetoc: 1` directive, now there is another one called `:hideindex: 1`, to hide the index in the left side.

**Comment by masaccio** [comment]
I will re-close and think on how I can use a TOC. I think I see a way of auto-generating both the Github README and the fuller docs from a single source.

Thanks for the prompt responses and the theme which is very much the balance of readability and highlighting important stuff that I wanted.

**Reply by danirus** [reply]
I am glad you like it :-)
